src/module.c

```diff
@@ -295,11 +295,11 @@ void ReplyWithSeriesLabels(RedisModuleCtx *ctx, const Series *series) {
 }
 
 void ReplyWithSeriesLastDatapoint(RedisModuleCtx *ctx, const Series *series) {
-    RedisModule_ReplyWithArray(ctx, 2);
     if(SeriesGetNumSamples(series) == 0) {
-        RedisModule_ReplyWithNull(ctx);
-        RedisModule_ReplyWithNull(ctx);
-    } else {
+        RedisModule_ReplyWithArray(ctx, 0);
```

Comment:
Lets return Null instead of empty list
```suggestion
        RedisModule_ReplyWithNull(ctx);
```